lib/model/model_wrapped.dart

```diff
@@ -104,11 +106,21 @@ class StudyModuleWrapped extends BaseWrapped {
 
   StudyModuleWrapped(BuildContext context, this._studyModule) : super(context);
 
+  String get term => _studyModule.when ?? "2020S"; // is this the best idea?
```

Comment:
I created a function `_currenTerm` in the service class, please see it [here](https://github.com/memoriesadrift/we-find/blob/main/lib/service/ufind_service.dart#L85). We could either create a `utils/utils.dart` file to hold that method (and maybe similar ones in the future) or you can just make it a public method by removing the leading underscore and import it directly. It will return the current term in a non-hardcoded way.